docs/cli/operator.mdx

```diff
@@ -564,6 +571,367 @@ Use the options listed below to change the command's behavior.
 
 - [--visibility-uri](/cli/cmd-options#visibility-uri)
 
+## nexus
+
+Nexus commands enable managing Nexus resources.
+
+Nexus commands follow this syntax:
+
+```
+temporal operator nexus [command] [command] [command options]
+```
+
+### endpoint
+
+Endpoint commands enable managing Nexus Endpoints.
+
+Endpoint commands follow this syntax:
+
+```
+temporal operator nexus endpoint [command] [command options]
+```
+
+#### create
+
+Create a new Nexus Endpoint on the Server.
```

Comment:
New isn't necessary because if you're creating an endpoint, it's new by default.